scripts/raydium/generate-config/main.go

```diff
@@ -0,0 +1,707 @@
+package main
```

Comment:
Im fine having this script in the repo but we should have defaults in the implementation as well

Reply:
Yeah I think it's probably necessary to have this somewhere. I hope it ends up being migrated to sauron pretty quickly though.

And I agree w/ David that we should either include a bunch of defaults in `/config/local/market.json` or have a separate config like `/config/raydium/market.json` where we can just run tests on raydium prices. Not sure what the ideal setup is there since we still need to integrate things into our dev/nightly tests more.

Reply:
Maybe there is some set of static values produced by this that we can include into `slinky-config` generation without having to pull in all of the node querying stuff?

Reply:
I'm a bit confused on what the slinky-config script does? Seems like it just generates a test config for the oracle, and for dydx it generates the set of markets  + oracle json it needs to normal operation?

Reply:
Since we aren't using this provider in prod, yet, I figured it was easiest to test w/ a separate config for raydium in particular. I can move to integrating this config into our own setup when I configure the dev slinky deployment